perfkitbenchmarker/test_util.py

```diff
@@ -66,3 +69,39 @@ def assertSampleListsEqualUpToTimestamp(self, a, b, msg=None):
         ex.message = ex.message + (' (was item %s in list)' % i)
         ex.args = (ex.message,)
         raise ex
+
+
+def assertDiskMounts(benchmark_config, mount_point):
+  """Test whether a disk mounts in a given configuration.
+
+  Sets up a virtual machine following benchmark_config and then tests
```

Comment:
benchmark_config may specify more than one VM, so I think it should be clear that only one VM from the config will be used. I think the best approach here is to force this function to only accept a spec with one VM (doc & assert) because the order of VMs in the spec.vms list appears to be random, so it's unclear which settings are used if more than one VM are specified.